quickwit-storage/src/object_storage/s3_compatible_storage.rs

```diff
@@ -55,6 +55,12 @@ use crate::retry::{retry, IsRetryable, Retry};
 use crate::{PutPayload, Storage, StorageErrorKind};
 use crate::{StorageError, StorageResult};
 
+/// A credential timeout.
+const CREDENTIAL_TIMEOUT: u64 = 5;
+
+/// An timeout for idle sockets being kept-alive.
+const POOL_IDIE_TIMEOUT: u64 = 10;
```

Comment:
typo in variable name `POOL_IDLE_TIMEOUT`

Reply:
Oh, sorry. I made a typo...
I fixed it.